spec/system/kudo_system_spec.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'Kudo management', type: :system do
+  before do
+    driven_by(:rack_test)
+  end
+
+  # rubocop:disable RSpec/ExampleLength
+
+  it 'enables me to create, edit and destroy kudos' do
+    employee = create(:employee)
+    kudo = create(:kudo)
```

Comment:
For the sake of consistency I suggest moving those outside the example, and use `let`
Also as with an employee - I would use `build` for kudo as well

Reply:
I used `let()` as suggested. Please take a look at it now. I think it looks a little clearer now ;)